exonum/src/node/mod.rs

```diff
@@ -448,6 +450,7 @@ impl NodeHandler {
             is_enabled,
             node_role,
             config_manager,
+            allow_expedited_propose: true,
```

Comment:
Should it be enabled at the very beginning?

Reply:
Why we never reset `allow_expedited_propose`?

Reply:
It is reseted in `maybe_add_propose_timeout`.

Reply:
This flag allows us to schedule a Propose timeout with minimal delay once when we see that we have enough transactions bunched up in the pool. It is set to `false` when we actually schedule this timeout (in `maybe_add_propose_timeout`) to prevent additional 'expedited' timeouts from being scheduled by that function until we handle this one. The flag is reset in `handle_propose_timeout` when we handle the propose timeout and are ready for the next one. It is initialized to `true` by default so that we can schedule a faster timeout from the get go.